benchmark/benchmark.py

```diff
@@ -79,36 +117,42 @@ def _mount_mp(
         subprocess_args.append(f"--max-threads={cfg['fuse_threads']}")
 
     log.info(f"Mounting S3 bucket {bucket} with args: %s; env: %s", subprocess_args, subprocess_env)
-    metadata["mount_s3_command"] = " ".join(subprocess_args)
-    metadata["mount_s3_env"] = subprocess_env
-    output = subprocess.check_output(subprocess_args, env=subprocess_env)
+    try:
+        output = subprocess.check_output(subprocess_args, env=subprocess_env)
+    except subprocess.CalledProcessError as e:
+        log.error(f"Error during mounting: {e}")
+        raise MountError()
```

Comment:
This is probably what you want: `raise MountError() from e`

Reply:
I think its implicit but updating it to be explicit anyway